lib/get_all.js

```diff
@@ -101,8 +101,42 @@ module.exports = {
     },
 
     getView: function(member, request, progressCallback) {
+        var pipeline = 5;
+        var pipelinePost = this.pipelinePost;
         return util.appFramework.promise(function(done, fail) {
-            util.getRows(member, request, done, fail, [], progressCallback);
+            util.appFramework.ajax.apply(util.appFramework, request).done(function(data) {
+                if(progressCallback) {
+                    progressCallback(data, data[member], request);
+                }
+                var pages = Math.ceil(data.count / 100);
+                var sectionLength = pages / pipeline;
+                if(pages < 2) {
+                    done(data[member]);
+                } else {
+                    var additional_pages = [];
+                    for (var i = 2; i <= pages; i++) {
+                        additional_pages.push(i);
+                    }
+                    var sections = util.splitArray(additional_pages, sectionLength);
+                    var requests = [];
+                    var final_data = data[member];
+                    var one_section = function(section) {
+                        return util.appFramework.promise(function(done, fail) {
+                            var local_request = request;
+                            local_request[2] = section.pop();
+                            util.getRows(member, request, section, done, fail, [], progressCallback);
+                        });
+                    };
+                    var concat_data = function(data) {final_data = final_data.concat(data);};
+
+                    sections.forEach(function(section) {
+                        requests.push(one_section(section).done(concat_data).fail(fail));
```

Comment:
Line is too long.<br>Identifier 'one_section' is not in camel case.<br>Identifier 'concat_data' is not in camel case.